File: scratch/InspectUsers.java
package scratch;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.Statement;

public class InspectUsers {
    public static void main(String[] args) {
        String url = "jdbc:sqlite:app-backend/vuavuive_v2.db";
        try {
            Class.forName("org.sqlite.JDBC");
            try (Connection conn = DriverManager.getConnection(url);
                 Statement stmt = conn.createStatement()) {
                
                System.out.println("--- USERS TABLE ---");
                String query = "SELECT id, full_name, email, phone, role, password_hash, is_active FROM users";
                try (ResultSet rs = stmt.executeQuery(query)) {
                    while (rs.next()) {
                        System.out.printf("ID: %s | Name: %s | Email: %s | Phone: %s | Role: %s | PasswordHash: %s | Active: %b%n",
                                rs.getString("id"),
                                rs.getString("full_name"),
                                rs.getString("email"),
                                rs.getString("phone"),
                                rs.getString("role"),
                                rs.getString("password_hash"),
                                rs.getBoolean("is_active"));
                    }
                }
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: scratch/inspect_users.py
import sqlite3
import json

conn = sqlite3.connect('app-backend/vuavuive_v2.db')
cursor = conn.cursor()
cursor.execute("SELECT id, full_name, email, phone, role, password_hash, is_active FROM users")
rows = cursor.fetchall()
users = []
for r in rows:
    users.append({
        "id": r[0],
        "full_name": r[1],
        "email": r[2],
        "phone": r[3],
        "role": r[4],
        "password_hash": r[5],
        "is_active": bool(r[6])
    })

print(json.dumps(users, indent=2, ensure_ascii=False))
conn.close()
